### ISSUE TITLE
Won't start.

### ISSUE THREAD

**Issue by Tayergo** [callum-oakley/gotta-go-fast#16]
It wasn't starting from normal terminal command, so I tried to manually start it via executable, which is in `/home/myuser/.local/bin/gotta-go-fast`, and it gave the error:
`gotta-go-fast: src/Main.hs:90:16-79: Missing field in record construction fillStrategy`

**Comment by MauricioRobayo** [comment]
Facing the same issue, installed with `stack install gotta-go-fast`, when executing `gotta-go-fast` I'm getting the same error:

```
gotta-go-fast: src/Main.hs:90:16-79: Missing field in record construction fillStrategy
```

I also downloaded source for [0.3.0.6](https://github.com/callum-oakley/gotta-go-fast/releases/tag/0.3.0.6) and tried building with `stack build` but got:

```
ghc: panic! (the 'impossible' happened)
  (GHC version 8.8.3 for x86_64-unknown-linux):
	Prelude.chr: bad argument: 1526726659

Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug


--  While building simple Setup.hs (scroll up to its section to see the error) using:
      /home/rfmaj/.stack/programs/x86_64-linux/ghc-tinfo6-8.8.3/bin/ghc-8.8.3 -rtsopts -threaded -clear-package-db -global-package-db -hide-all-packages -package base -main-is StackSetupShim.mainOverride -package Cabal-3.0.1.0 /home/rfmaj/.stack/setup-exe-src/setup-mPHDZzAJ.hs /home/rfmaj/.stack/setup-exe-src/setup-shim-mPHDZzAJ.hs -o /home/rfmaj/.stack/setup-exe-cache/x86_64-linux-tinfo6/tmp-Cabal-simple_mPHDZzAJ_3.0.1.0_ghc-8.8.3
    Process exited with code: ExitFailure 1
```

Seems there is no way to currently get this working (I'm on Fedora 36).

**Comment by callum-oakley** [comment]
Thanks both for letting me know. Looks like a dependency has changed and stack is using the newer version when you stack install. I thought stack was supposed to pin to a specific snapshot of all the dependencies... but maybe I've configured something wrong. Will see if I can figure out what needs to be pinned when I get a mo, watch this space... 😅  

> The primary stack design point is reproducible builds. If you run stack build today, you should get the same result running stack build tomorrow.

🤔

**Comment by callum-oakley** [comment]
Okay, this should be fixed in 0.3.0.8.

```
stack update
stack install gotta-go-fast
```
